Makefile

```diff
@@ -36,6 +34,23 @@ GOBIN ?= $(GOPATH)/bin
 DOCKER_CLI_EXPERIMENTAL = enabled
 export GOPATH GOBIN GO111MODULE DOCKER_CLI_EXPERIMENTAL
 
+# Generate all combination of all OS, ARCH, and OSVERSIONS for iteration
+ALL_OS = linux windows
+ALL_ARCH.linux = amd64
+ALL_OS_ARCH.linux = $(foreach arch, ${ALL_ARCH.linux}, linux-$(arch))
+ALL_ARCH.windows = amd64
+ALL_OSVERSIONS.windows := 1809 1903 1909 2004
```

Comment:
@AbelHu do we really need so many os versions on Windows? it's 4 versions now, and all these 4 versions are not compatible?

Reply:
in the beginning, there is only `1809`, so what's the other `1903 1909 2004` for? Are they all necessary?

Reply:
what about build `1809, 2019, 2004` as @AbelHu suggested?

Reply:
@marosset could you comment on this?

Reply:
None of the 4 versions listed here are compatible with each other. 
Let's keep all 4 of 1809, 1903, 1909, and 2004 for now so we'll include a version for each Windows Server version starting with Windows Server 2019.
This matches what gets built for the test and pause images in k/k
https://github.com/kubernetes/kubernetes/blob/ac996a37f65a8f11640808a883e9522832ed364f/build/pause/Makefile#L39